### ISSUE TITLE
Have a "helpful hints" expandable section at upload screen

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#1647]
There are a few inner workings of the app that are not immediately obvious to the casual user, I think. A few off the top of my head:

- Enabling geotagging on your camera is necessary to allow the app to attach location templates to your uploaded image and provide location-based category suggestions 
- If you mistakenly upload a wrong image, you can nominate it for deletion within the app by going to the media details view and selecting "nominate for deletion"
- Category selection is very important to ensure that your picture is usable in Wikipedia articles, because that is how people usually look for images to add to articles

So on and so forth. For desktop apps and games, a common tactic is to display "helpful hints" on the loading screen. We have no such option, so I figured that having an expandable section on the upload screen (or perhaps even underneath a contribution that is currently being uploaded) might be a good way of getting these across.

The main consideration IMO is that we don't want to clutter up our UI too much, and also that we have to maintain this to ensure that it keeps up with changes in the app.

**Comment by misaochan** [comment]
Alternatively, we could display a random hint on the upload screen each time, and have a "see all" option that would bring up the full list?

**Comment by misaochan** [comment]
@ilgazer I agree, one hint at a time would be best. But I guess the "see all" option is useful - if a user REALLY wants to read all of them at the same time, I don't see why not? :)

**Comment by chrisdebian** [comment]
> @ilgazer I agree, one hint at a time would be best. But I guess the "see all" option is still useful - if a user REALLY wants to read all of them at the same time, I don't see why not? :)

This sounds like a good idea, do we know if anyone is able to implement it?

Chris
